### ISSUE TITLE
Hip adductors weakness

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5932]
**Preferred term label:** Hip adductors weakness

**Synonyms** Weak hip adductors

**Definition (free text, please give PubMed ID)** Reduced ability to bring the leg toward the midline of the body., PMID:28914719

**Parent term (use hpo.jax.org/app)** Pelvic girdle muscle weakness HP:0003749

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
